setup.py

```diff
@@ -19,23 +20,24 @@
 have_cython = False
 cmdclass = {}
 if platform in ('android', 'ios'):
-    print 'Cython import ignored'
+    print('Cython import ignored')
 else:
     try:
         from Cython.Distutils import build_ext
         have_cython = True
         cmdclass['build_ext'] = build_ext
     except ImportError:
-        print '**** Cython is required to compile audiostream ****'
+        print('**** Cython is required to compile audiostream ****')
         raise
 
 # configure the env
 use_sdl2 = environ.get('USE_SDL2')
 if use_sdl2:
-    include_dirs = ['/usr/include/SDL2']
```

Comment:
any particular reason for this?

Reply:
The "immintrin.h" header was requested by "SDL_cpuinfo.h" located in /usr/include/SDL2 .

When compiling via the python-for-android "audiostream" recipe, this causes an error.

I think that at least for the Android release, we have to refer to the SDL2_INCLUDE_DIR env variable.

I'm appending it later as a fallback. (Not sure if is needed).

Reply:
If it works, this seems okay to me. It makes sense that the env var should have priority.

Reply:
@inclement , tested on Android w/ python3 and it worked.